.travis.yml

```diff
@@ -21,207 +22,207 @@ addons:
-    - PEP8=false TESTCASE=pythran/tests/test_spec_parser.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-g"
-    - PEP8=false TESTCASE=pythran/tests/test_spec_parser.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-DUSE_BOOST_SIMD -march=native"
-    - PEP8=false TESTCASE=pythran/tests/test_spec_parser.py CC=gcc-5 CXX=g++-5 CXXFLAGS="-fopenmp"
-    - PEP8=false TESTCASE=pythran/tests/test_spec_parser.py CC=gcc-5 CXX=g++-5 CXXFLAGS="-DUSE_BOOST_SIMD -march=native -fopenmp"
-    - PEP8=false TESTCASE=pythran/tests/test_str.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-g"
-    - PEP8=false TESTCASE=pythran/tests/test_str.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-DUSE_BOOST_SIMD -march=native"
-    - PEP8=false TESTCASE=pythran/tests/test_str.py CC=gcc-5 CXX=g++-5 CXXFLAGS="-fopenmp"
-    - PEP8=false TESTCASE=pythran/tests/test_str.py CC=gcc-5 CXX=g++-5 CXXFLAGS="-DUSE_BOOST_SIMD -march=native -fopenmp"
-    - PEP8=false TESTCASE=pythran/tests/test_submodules.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-g"
-    - PEP8=false TESTCASE=pythran/tests/test_submodules.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-DUSE_BOOST_SIMD -march=native"
-    - PEP8=false TESTCASE=pythran/tests/test_submodules.py CC=gcc-5 CXX=g++-5 CXXFLAGS="-fopenmp"
-    - PEP8=false TESTCASE=pythran/tests/test_submodules.py CC=gcc-5 CXX=g++-5 CXXFLAGS="-DUSE_BOOST_SIMD -march=native -fopenmp"
-    - PEP8=false TESTCASE=pythran/tests/test_time.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-g"
-    - PEP8=false TESTCASE=pythran/tests/test_time.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-DUSE_BOOST_SIMD -march=native"
-    - PEP8=false TESTCASE=pythran/tests/test_time.py CC=gcc-5 CXX=g++-5 CXXFLAGS="-fopenmp"
-    - PEP8=false TESTCASE=pythran/tests/test_time.py CC=gcc-5 CXX=g++-5 CXXFLAGS="-DUSE_BOOST_SIMD -march=native -fopenmp"
-    - PEP8=false TESTCASE=pythran/tests/test_typing.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-g"
-    - PEP8=false TESTCASE=pythran/tests/test_typing.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-DUSE_BOOST_SIMD -march=native"
-    - PEP8=false TESTCASE=pythran/tests/test_typing.py CC=gcc-5 CXX=g++-5 CXXFLAGS="-fopenmp"
-    - PEP8=false TESTCASE=pythran/tests/test_typing.py CC=gcc-5 CXX=g++-5 CXXFLAGS="-DUSE_BOOST_SIMD -march=native -fopenmp"
-    - PEP8=false TESTCASE=pythran/tests/test_user_defined_import.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-g"
-    - PEP8=false TESTCASE=pythran/tests/test_user_defined_import.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-DUSE_BOOST_SIMD -march=native"
-    - PEP8=false TESTCASE=pythran/tests/test_user_defined_import.py CC=gcc-5 CXX=g++-5 CXXFLAGS="-fopenmp"
-    - PEP8=false TESTCASE=pythran/tests/test_user_defined_import.py CC=gcc-5 CXX=g++-5 CXXFLAGS="-DUSE_BOOST_SIMD -march=native -fopenmp"
-    - PEP8=false TESTCASE=pythran/tests/test_format.py CC=gcc-5 CXX=g++-5
-    - PEP8=true
+    - PYTHON_VERSION=2 PEP8=false TESTCASE=pythran/tests/test_advanced.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-g"
```

Comment:
> > +    - PYTHON_VERSION=2 PEP8=false TESTCASE=pythran/tests/test_advanced.py CC=clang-3.5 CXX=clang++-3.5 CXXFLAGS="-g"
> 
> You only test Python2?

You can see some PYTHON_VERSION=3, in that case both PYTHON2 and PYTHON3
are tested